test/Passes/xsmm-combine.mlir

```diff
@@ -0,0 +1,254 @@
+
+memref.global "private" constant @__constant_4x32x32xf32 : memref<4x32x32xf32> = dense<1.000000e+00> {alignment = 128 : i64}
+memref.global "private" constant @__constant_8x32x32xf32 :  memref<8x32x32xf32> = dense<1.000000e+00> {alignment = 128 : i64}
+memref.global "private" constant @__constant_32xf32:  memref<32xf32, strided<[32], offset:?>> = dense<1.000000e+00> {alignment = 128 : i64}
+
+func.func @bcast_col_in0_on_binary_add(%arg0: memref<256x128xf32>) -> memref<256x512xf32>  {
+ %c0 = arith.constant 0 : index
+  %c8 = arith.constant 8 : index
+  %c4 = arith.constant 4 : index
+  %c1 = arith.constant 1 : index
+  %c16 = arith.constant 16 : index
+  %c4_i64 = arith.constant 4 : i64
+  %c8_i64 = arith.constant 8 : i64
+  %cst = arith.constant 0.000000e+00 : f32
+  %0 = memref.get_global @__constant_4x32x32xf32 : memref<4x32x32xf32>
+  %1 = memref.get_global @__constant_8x32x32xf32 : memref<8x32x32xf32>
+  %2 = memref.get_global @__constant_32xf32 : memref<32xf32, strided<[32], offset:?>>
+  %alloc = memref.alloc() {alignment = 64 : i64} : memref<8x4x32x32xf32>
+  %3 = xsmm.unary.dispatch identity [32, 32, 128, 32] flags = (none) data_type = f32
+  %alloc_0 = memref.alloc() {alignment = 64 : i64} : memref<8x8x32x32xf32>
+  %4 = xsmm.brgemm.dispatch [32, 32, 32, 32, 32, 32, 1024, 1024] flags = (beta_0) data_type = f32
+  %5 = xsmm.binary.dispatch add [32, 32, 32, 32, 32] flags = (bcast_col_in0) data_type = f32
+  %6 = xsmm.unary.dispatch relu [32, 32, 32, 32] flags = (none) data_type = f32
+  %alloc_1 = memref.alloc() {alignment = 64 : i64} : memref<256x512xf32>
+  scf.parallel (%arg3, %arg2) = (%c0, %c0) to (%c8, %c8) step (%c1, %c1) {
+	%subview = memref.subview %alloc_0[%arg3, %arg2, 0, 0] [1, 1, 32, 32] [1, 1, 1, 1] : memref<8x8x32x32xf32> to memref<32x32xf32, strided<[32, 1], offset: ?>>
+	%subview_2 = memref.subview %alloc[%arg3, 0, 0, 0] [1, 4, 32, 32] [1, 1, 1, 1] : memref<8x4x32x32xf32> to memref<4x32x32xf32, strided<[1024, 32, 1], offset: ?>>
+	xsmm.brgemm(data_type = f32, %4, %subview_2, %0, %subview, %c4_i64) : (i64, memref<4x32x32xf32, strided<[1024, 32, 1], offset: ?>>, memref<4x32x32xf32>, memref<32x32xf32, strided<[32, 1], offset: ?>>, i64) -> ()
+	xsmm.binary add(data_type = f32, %5, %2, %subview, %subview) : (i64, memref<32xf32, strided<[32], offset:?>>, memref<32x32xf32, strided<[32,1], offset: ?>>, memref<32x32xf32, strided<[32, 1], offset: ?>>) -> ()
+	xsmm.unary relu(data_type = f32, %6, %subview, %subview) : (i64, memref<32x32xf32, strided<[32, 1], offset: ?>>, memref<32x32xf32, strided<[32, 1], offset: ?>>) -> ()
+	scf.yield
+  }
+  return %alloc_1 : memref<256x512xf32>
+ }
+
+// CHECK-LABEL: func.func @bcast_col_in0_on_binary_add(
+// CHECK: %[[ARG0:.*]]: memref<256x128xf32>) -> memref<256x512xf32> {
+// CHECK: %[[dispatch:.*]] = xsmm.fused_brgemm.dispatch [32, 32, 32, 32, 32, 32, 1024, 1024][add,relu]  flags = (none)  binary_flags = (bcast_col_in0)  unary_flags = (none) data_type = f32
+// CHECK: xsmm.fused_brgemm
```

Comment:
Add an CHECK line for %2 and make sure it's in the right order on the invoke:
```mlir
  CHECK: %[[BIAS:.*]] = memref.get_global {{.*}} memref<32xf32>
  ...
  CHECK: xsmm.fused_brgemm(data_type = f32, [[dispatch]], {{.*}}, {{,.*}}, {{.*}}, [[BIAS]]) 
```

Same below with `col_in1` etc to make sure you got the right bias to fuse.